### PR TITLE
Fix some indenting, add JMM Anime and Sonarr bundles.

### PR DESCRIPTION
**Pull request by CatmanIX**
s/tabs/spaces
added: https://github.com/japanesemediamanager/plexplugin
added: https://github.com/jamorin/Sonarr.bundle

### CODE REVIEW

File: Resources/plugin_details.json [hunk 1]
```diff
@@ -615,7 +615,18 @@
...
+    {
+        "title"           :     "My Anime",
+        "bundle"          :     "MyAnime.bundle",
+        "type"            :     ["Video"],
+        "description"     :     "JapaneseMediaManager Plex plugin.",
+        "repo"            :     "https://github.com/japanesemediamanager/plexplugin",
+        "branch"          :     "master",
+        "identifier"      :     "com.plexapp.plugins.myanime",
+        "icon"            :     "icon-myanime.png",
+        "supporturl"      :     "https://forums.plex.tv/discussion/81430"
```
Comment (Twoure) [inline Resources/plugin_details.json]: This is the support URL for _MTV Shows_, not _My Anime_.
https://forums.plex.tv/discussion/91096 looks to be the original support forum.

**Comment by CatmanIX** [comment]
@Twoure thanks, missed that, fixed.

**Comment by ukdtom** [comment]
Are you the developer of those?

Cuz if not, I can't add them.

And the support links are for Github.
I need them for the channel forum, as laid out in the wiki, since regular users will NOT go to github, and then end up in the UAS thread instead, which I do NOT want

And if you are not the dev, then create an issue in the gits, and ask the dev to add them

/T

**Comment by CatmanIX** [comment]
I'm not the dev, no, i don't really see why it needs to be the developer who adds them but I'll get in touch with them i guess. If they just throw their approval in this pull request would that be fine? I don't know if the sonarr bundles dev is still active so i might adopt that project if i can't get in touch.

Most of the plugins don't even have support URLs at the moment and half the ones that do are github issue pages. Is no supporturl better than a github supporturl? You might want to tell the trakt guys pull request that too. My Anime has a plex page, i can add that, but i don't think the sonarr bundle does.

**Comment by ukdtom** [comment]
Well, In my book, the owner is the one that makes the call, since I do not want to get in trouble, in case the dev didn't wanted it on the UAS.

And as said, plex support forum support thread is mandatory, and reason is simply, that the old uas thread had been read by +1.6 million times, was the busiest thread up there, but sadly, less than 10 % of the posts in there was related to the uas, and the rest to other plugin's.
I simply do not have the time for a thread like that!

**Comment by ukdtom** [comment]
Closing for now....Devs should create a new PR

**Pull request closed without merging**